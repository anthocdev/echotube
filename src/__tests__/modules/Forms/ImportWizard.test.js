import React from "react";
import ReactDOM from "react-dom";
import ImportWizard from "../../../modules/containers/Forms/VideoImportWizard/ModalWindow";

/* @todo: set up basic video object for use in test */

it("Successfully renders", () => {
  const div = document.createElement("div");

  ReactDOM.render(<ImportWizard playlistId={1}></ImportWizard>, div);
});
