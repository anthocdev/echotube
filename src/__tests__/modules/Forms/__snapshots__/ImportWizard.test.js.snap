// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Import Wizard Module Test Renders Matches Snapshot 1`] = `
<div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "backgroundColor": "rgba(255,255,255,0.7)",
        "marginBottom": "20px",
        "marginTop": "20px",
      }
    }
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Import a New Video
    </span>
    <span
      className="MuiTouchRipple-root"
    />
  </button>
</div>
`;
